configs/azure.reference.conf

```diff
@@ -920,7 +952,7 @@ cluster {
     # Optional override of CDH parcel repositories
     #
 
-    # parcelRepositories: ["http://archive.cloudera.com/cdh5/parcels/5.9/"]
+    # parcelRepositories: ["http://archive.cloudera.com/cdh5/parcels/5.5/"]
```

Comment:
@jasonbw I think this needs to be reverted. Same comment for the other config samples.